docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+## API Documentation
 
-# API calls
+Chatbot_ner designed to extract essential entities or information like date, time, numbers, texts (city, organisation name) from textual data. These APIs are built keeping in mind to plug it in for conversational AI applications.
 
-[TOC]
+Following are the different entity types we currently support. 
 
-## List of entity types
+| Entity Type                     | Description                                                  | Examples                                                     |
+| ------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Time](#1-time)                 | Detects time in various formats from given text.             | tomorrow morning at 5, कल सुबह ५ बजे, kal subah 5 baje         |
+| [Date](#2-date)                 | Detects date in various formats from given text.             | 28-12-2096, 09th Nov 2014, Tomorrow, next monday, agle somvar, अगले सोमवार |
+| [Number](#3-number)             | Detects number from the text.                                | 50 rs per person, ५ किलो चावल, मुझे एक लीटर ऑइल चाहिए          |
+| [Phone number](#4-phone-number) | Detects phone numbers in given text.                         | +919222222222                                                |
+| [Email](#5-email)               | Detects email addresses in given text.                       | amans.rlx@gmail.com                                          |
+| [Text](#6-text)                 | Detect custom entities in text string using full text search in Datastore or based on contextual model | **pizza**, **मुंबई**                                           |
+| [PNR-Number](#7-pnr-number)     | Detect PNR (serial) codes in given text.                     | My flight PNR is **4SGX3E**                                  |
+| [Regex](#8-regex)               | Detect entities using custom regex patterns                  | My flight PNR is **4SGX3E**<br />Please apply **CASH20** coupon code |
 
-Following are the list of different entity types along with its API call:
 
-### text
 
-- This functionality calls the TextDetection class to detect textual entities.
+### APIs Parameters
 
-- Example:
+Following are the list of parameters accepted by APIs used for detection:
 
-  - Example 1: 
+1. **message**: Message on which detection logic needs to run. It is an unstructured text from which entity needs to be extracted. For example, *"I want to order pizza"*.
 
-    - ```python
-      message='i want to order chinese from  mainland china and pizza from domminos'
```

Comment:
All values of `message`, `structured_value`, `fallback_value`, `bot_message` should have a `u` prefix as the shell examples are meant to be run in Python 2 shell and Non-English stuff might fail when it is encoded.

Same suggestion for outputs where out detectors return Unicode literals in outputs